workbench-core/datasets/src/dataSetService.ts

```diff
@@ -97,7 +95,7 @@ export class DataSetService {
    * @param dataSetId - the ID of the DataSet to remove.
    */
   public async removeDataSet(dataSetId: string): Promise<void> {
-    throw new Error(notImplementedText);
+    await this._dbProvider.removeDataSet(dataSetId);
```

Comment:
state that even if dataset does not exist, it will not throw

Reply:
done